doc/preprocessing.doc

```diff
@@ -259,6 +259,10 @@ you can run doxygen as follows:
 \verbatim
   doxygen -d Preprocessor
 \endverbatim
+or without line numbers:
+\verbatim
+  doxygen -d PreprocessorNolineno
```

Comment:
I think it would simplify things a bit if a `-d nolines` option is just added to the list, so without line numbers you'd call doxygen with `-d Preprocessor -d nolines`. Then the code below can just `|` the flag and in places where it makes a difference do a `Debug::isFlagSet(Debug::NoLines)`.

Reply:
No objections, should I update the PR or will you do it?

Reply:
If you can do it, then that would be nice 😃

Reply:
It is on my list for tomorrow